README.md

```diff
@@ -16,11 +16,11 @@ Please follow [this](https://www.botkube.io/installation/) for complete BotKube
 
 ## Architecture
 ![](/botkube_arch.jpg)
-- **Informer Controller:** Registers informers to kube-apiserver to watch events on the configured k8s resources. It forwards the incoming k8s event to the Event Manager
-- **Event Manager:** Extracts required fields from k8s event object and creates a new BotKube event struct. It passes BotKube event struct to the Filter Engine
+- **Informer Controller:** Registers informers to kube-apiserver to watch events on the configured k8s resources. It forwards the incoming k8s event to the Event Manager.
+- **Event Manager:** Extracts required fields from k8s event object and creates a new BotKube event struct. It passes BotKube event struct to the Filter Engine.
 - **Filter Engine:** Takes the k8s object and BotKube event struct and runs Filters on them. Each filter runs some validations on the k8s object and modifies the messages in the BotKube event struct if required.
 - **Event Notifier:** Finally, notifier sends BotKube event over the configured communication channel.
-- **Bot Interface:** Bot interface takes care of authenticating and managing connections with communication mediums like Slack, Mattermost. It reads/sends messages from/to communication mediums. 
```

Comment:
Can you please also add `Microsoft Teams` in addition to Slack, Mattermost across all the occurrences?

Reply:
@PrasadG193 Done